1-D_Array/Python/Frequency_Of_A_Element.py

```diff
@@ -0,0 +1,27 @@
+num = int(input("Enter number of elements in the array"))
```

Comment:
@divya1999ann there are 4 files in this PR. Can you please remove the files not related to this PR before we review this?